diff --git a/node_modules/preact-cli/lib/lib/entry.js b/node_modules/preact-cli/lib/lib/entry.js
index 92e2d61..bc566ad 100644
--- a/node_modules/preact-cli/lib/lib/entry.js
+++ b/node_modules/preact-cli/lib/lib/entry.js
@@ -14,7 +14,7 @@ if (process.env.NODE_ENV === 'development') {
 	// only add a debug sw if webpack service worker is not requested.
 	if (process.env.ADD_SW === undefined && 'serviceWorker' in navigator) {
 		// eslint-disable-next-line no-undef
-		navigator.serviceWorker.register(__webpack_public_path__ + 'sw-debug.js');
+		//navigator.serviceWorker.register(__webpack_public_path__ + 'sw-debug.js');
 	} else if (process.env.ADD_SW && 'serviceWorker' in navigator) {
 		// eslint-disable-next-line no-undef
 		navigator.serviceWorker.register(
